assembl/static2/js/app/main.jsx

```diff
@@ -23,33 +19,19 @@ type State = { identifier: string, location: string };
 class Main extends React.Component<Props, State> {
   constructor(props: Props) {
     super(props);
-    const { debateData } = this.props.debate;
-    const paramsIdentifier = this.props.params.phase || getCurrentPhaseIdentifier(debateData.timeline);
+    const { timeline } = this.props;
+    const paramsIdentifier = this.props.params.phase || getCurrentPhaseIdentifier(timeline);
     const queryIdentifier = this.props.location.query.phase || paramsIdentifier;
     this.state = {
       identifier: queryIdentifier,
       location: this.props.location.pathname
     };
   }
 
-  componentWillMount() {
-    const { debateData } = this.props.debate;
-    const currentPhaseIdentifier = getCurrentPhaseIdentifier(debateData.timeline);
-    let isRedirectionToV1;
-    if (debateData.timeline === null) {
-      // timeline is not configured
-      isRedirectionToV1 = true;
-    } else {
-      const currentPhase = debateData.timeline.filter(phase => phase.identifier === currentPhaseIdentifier);
-      isRedirectionToV1 = currentPhase[0] && currentPhase[0].interface_v1;
-    }
-    this.props.addRedirectionToV1(isRedirectionToV1);
-  }
-
   componentWillReceiveProps(nextProps) {
```

Comment:
`componentWillReceiveProps` will be deprecated, it could be a good occasion to change it here